### ISSUE TITLE
LIR Name=Enge return outdated entry, without PtMode

### ISSUE THREAD

**Issue by vasile** [openTdataCH/ojp-sdk#27]
https://tools.odpch.ch/ojp-api-explorer-v2/?gist=cebb74c9fbacc0ed3f4274769a947bae

2nd entry is `Zürich Enge (See)`, DIDOK `8503449` and doesnt have a PtMode content
Reason is because the entry is outdated, was discontinued in 2021?
https://atlas.app.sbb.ch/service-point-directory/service-points/8503449/service-point

```
<PlaceResult>
  <Place>
    <StopPlace>
      <StopPlaceRef>8503449</StopPlaceRef>
      <StopPlaceName>
        <Text xml:lang="de">Zürich Enge (See)</Text>
      </StopPlaceName>
      <PrivateCode>
        <System>EFA</System>
        <Value>108608</Value>
      </PrivateCode>
      <TopographicPlaceRef>23026261:27</TopographicPlaceRef>
    </StopPlace>
    <Name>
      <Text xml:lang="de">Zürich Enge (See) (Zürich)</Text>
    </Name>
    <GeoPosition>
      <siri:Longitude>8.53717</siri:Longitude>
      <siri:Latitude>47.35913</siri:Latitude>
    </GeoPosition>
  </Place>
  <Complete>true</Complete>
  <Probability>0.91</Probability>
</PlaceResult>
```

**Comment by vasile** [comment]
Maybe a clue is that in the LIR BBOX query the entry doesnt show up:

| left | right |
|-|-|
| atlas screenshot with the old position (pink)  |  OJP 2.0 LIR BBOX with `stop` filter |

![image](https://github.com/openTdataCH/ojp-sdk/assets/113994/352ca527-b2a2-4827-9946-4a1d881ea1f2)

**Comment by TO-mdv** [comment]
Need to clean up data. There are currently 5800 not used Stops in the system. Will clean on TEST, and if it's ok, I will do it on INT/PROD as well.

**Comment by TO-mdv** [comment]
Cleanup data, deleted 5800 not used stops. The cleaned data's are available on PROD on 10.6.2024 at 15:00

**Comment by TO-mdv** [comment]
Released on PROD 10.6.2024
